src/github/server.rs

```diff
@@ -90,11 +91,14 @@ pub fn create_bors_process(
     mpsc::Sender<BorsGlobalEvent>,
     impl Future<Output = ()>,
 ) {
+    let ctx = Arc::new(ctx);
+
     let (repository_tx, repository_rx) = mpsc::channel::<BorsRepositoryEvent>(1024);
     let (global_tx, global_rx) = mpsc::channel::<BorsGlobalEvent>(1024);
+    let mut mergeable_queue = MergeableQueue::new(ctx.clone());
 
     let service = async move {
-        let ctx = Arc::new(ctx);
+        mergeable_queue.start();
```

Comment:
Any particular reason why you use `tokio::spawn` vs just returning a `Future` that we could poll here? I prefer that, because it makes it simpler to track the execution of the future, and also make sure that it stops together with the `service` future.